src/opm/output/eclipse/Summary.cpp

```diff
@@ -971,28 +971,50 @@ inline quantity potential_rate( const fn_args& args ) {
 
 inline quantity preferred_phase_productivty_index(const fn_args& args) {
     if (args.schedule_wells.empty())
-        return potential_rate<rt::productivity_index_oil>(args);
+        return {0, rate_unit<rt::productivity_index_oil>()};
```

Comment:
Strictly speaking, `potential_rate` will return the appropriate zero value in this case.

Reply:
I agree - but when we were already returning explicit zeros I think it became clearer this way. No strong feelings though.